lib/mutant/env/bootstrap.rb

```diff
@@ -95,7 +95,7 @@ def scope_name(scope)
       # @api private
       def infect
         config.includes.each(&$LOAD_PATH.method(:<<))
-        config.requires.each(&method(:require))
+        config.requires.each(&Kernel.method(:require))
```

Comment:
Actually I think next refactorings will pass in Kernel, $stderr, $stdout and other global friends explicitly into the mutant namespace. I had great success with that on the `Mutant::Zombifier` sub-project. So it'll help everywhere.

Each time I have to place a message expectation via rspec on a global constant something in my heart dies.